tools/flaky_test_monitor/testdata/summary2/test4-multi-failures/expected-output/failures/TestSanitySha2_256/failure9.txt

```diff
@@ -0,0 +1,16 @@
+=== RUN   TestSanitySha2_256
```

Comment:
I'm super confused, these all look the same? Why can't we just re-use?

Reply:
We are simulating the scenario where the test has failed 10 times and we want to examine the failure message of each failure.

In other words, these are individual failure messages for the same test over many test runs. 

In this case they all have the same failure message across multiple failures but that doesn't always have to be the case - there could be different failure messages for the same test if the test is flaky in multiple places.

We need to save each failed test output so it can be further examined by the test owner.